### ISSUE TITLE
Kymobutler issue

### ISSUE THREAD

**Issue by Melody-subra** [fabricecordelieres/IJ-Plugin_KymoButler_for_ImageJ#6]
I got an error message when I run the program: "the response doesnt seems to be formated properly"

Suprisingly it was working well 1 month ago. I re-installed the plugin but it is still showing the same error message.

My folders are in the right file.

Hopefully someone experienced the same issue here.

Best,
Melody

**Comment by carmenp3** [comment]
Same issue here

**Comment by fabricecordelieres** [comment]
This behavior is not due to the plugin.
Some abuses have been reported about the use of the webportal "free version", leading to some additional costs for the company holding rights to the KymoButler algorithm/implementation. As a consequence, the portal has been shut down. I've contacted DeepMirror and don't know yet what the follow-up will be.
This plugin I wrote was only meant to make the link between ImageJ and the webportal. There is unfortunately nothing I can do for now, from my side.
